script/helperFunctions.sh

```diff
@@ -1357,16 +1357,16 @@ function verifyContract() {
     if [ "$ARGS" = "0x" ]; then
       # only show output if DEBUG flag is activated
       if [[ "$DEBUG" == *"true"* ]]; then
-        forge verify-contract --watch --chain "$NETWORK" "$ADDRESS" "$FULL_PATH" -e "${!API_KEY}"
+        forge verify-contract --watch --chain "$CHAIN_ID" "$ADDRESS" "$FULL_PATH" -e "${!API_KEY}"
```

Comment:
Why only change it here?
Was there an issue?
Should we switch to --chain throughout all scripts?

Reply:
It should already be $CHAIN_ID throughout. I just did a search.